src/app/app.component.ts

```diff
@@ -0,0 +1,70 @@
+import { Component, OnInit } from '@angular/core';
+import { ActivatedRoute, NavigationEnd, Router } from '@angular/router';
+import { filter } from 'rxjs/operators';
+import { Title } from '@angular/platform-browser';
+import { LocalDataService } from './shared/services/local-data.service';
+import { User } from './shared/models/user.model';
+import { AuthService } from './shared/services/auth.service';
+import { UserBlockService } from './shared/services/user-block.service';
+
+@Component({
+  selector: 'app-root',
+  templateUrl: './app.component.html',
+  styleUrls: ['./app.component.scss']
+})
+export class AppComponent implements OnInit {
+  public show: Boolean = true;
```

Comment:
According to the task requirenments app component shouldn't have any logic

![image](https://user-images.githubusercontent.com/33813087/93913965-7e62e000-fd0e-11ea-85e8-96f1d7c868b7.png)